src/ngMock/angular-mocks.js

```diff
@@ -1382,14 +1382,14 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
         var response = wrapped.response(method, url, data, headers, wrapped.params(url));
         xhr.$$respHeaders = response[2];
         callback(copy(response[0]), copy(response[1]), xhr.getAllResponseHeaders(),
-                 copy(response[3] || ''));
+               copy(response[3] || ''), copy(response[4]));
```

Comment:
Well, the linter agrees so ... (on a serious note: no clue, I'll get it removed before merging)